### PR TITLE
added report on volunteers who are owed refunds

### PR DESCRIPTION
**Pull request by EliAndrewC**
In the past we generally waited for someone to come to Stops and ask for a refund.  This used to be sensible since most of the time their hours wouldn't be marked until they brought us their sheet anyway.

Nowadays with people having smartphones and network access being more prevalent, we can refund volunteer registrations as soon as their hours are marked off, so it's nice to have a report so we can start doing that during quiet time on Saturday night.

### CODE REVIEW

File: uber/configspec.ini [hunk 1]
```diff
@@ -91,12 +91,18 @@ untransferable_attrs = string_list(default=list('first_name','last_name','email'
 # a list of department constants you want to be excluded from the shift system.
 shiftless_depts = string_list(default=list())
```
Comment (EliAndrewC) [inline uber/configspec.ini]: I figured it was worth making this configurable, even if it probably won't change for any MAGFest-run event.

File: uber/configspec.ini [hunk 2]
```diff
@@ -91,12 +91,18 @@ untransferable_attrs = string_list(default=list('first_name','last_name','email'
...
+# informational pages and reports.
+hours_for_refund = integer(default=24)
+
 # There are two separate deadlines for custom badges; the one after which it's
 # too late for attendees to edit their custom badge submissions, and the one
 # where it's too late for an admin to shift badge numbers on the backend.  The
 # former is controlled by the PRINTED_BADGE_DEADLINE field, which is what we
 # advertise in our emails and such.
-
+#
```
Comment (EliAndrewC) [inline uber/configspec.ini]: Unrelated style fix.

File: uber/templates/static_views/stafferComps.html [hunk 3]
```diff
@@ -11,7 +11,7 @@ <h3 align="center"> What You Get For Volunteering </h3>
     <li> Working at least 6 hours gets you a {{ c.EVENT_NAME }} t-shirt. <br/><br/> </li>
     <li> Working at least 12 hours gets you food in addition to the shirt.  We have professional chefs preparing 3 meals per day throughout the event. <br/><br/> </li>
     <li> Working at least 18 hours gets you a complementary Staff badge for next year's {{ c.EVENT_NAME }}, which will make you eligible for space in one of our staff hotel rooms. <br/><br/> </li>
-    <li> Working at least 24 hours gets you a refund on your badge for this year. <br/><br/> </li>
+    <li> Working at least {{ c.HOURS_FOR_REFUND }} hours gets you a refund on your badge for this year. <br/><br/> </li>
```
Comment (EliAndrewC) [inline uber/templates/static_views/stafferComps.html]: Surprisingly, this was the only other place in the code where we referred to the number 24 w.r.t. hour requirements.

**Comment by binary1230** [comment]
+!
